src/components/common/Grid.js

```diff
@@ -0,0 +1,19 @@
+import styled  from 'styled-components';
+import { devices } from '../../utils/devices';
+
+const Grid = styled.div`
```

Comment:
Normally the styled-components add the prefix `Styled` so it's easily identified by other people. In this case it could be `StyledGrid`.
Also, these kinds of components should be in a dedicated file `styled.js`, the name `Grid.js` it's confusing, we don't know if the component it's part of the app as a container or representational.